integration/sql/README.md

```diff
@@ -24,6 +24,11 @@ curl https://sh.rustup.rs -sSf | sh -s -- -y
 source $HOME/.cargo/env
 ```
 
+Make sure you are in the correct directory.
+```bash
+cd iface-py
+```
+
```

Comment:
It's right, but those lines should be after `cargo test`. You should run tests from root directory.